nextflow.config

```
@@ -242,21 +242,85 @@ manifest {
             orcid: '0009-0001-9875-5262'
         ],
         [
-            name: ' Susanne Jodoin',
-            affiliation: '	Quantitative Biology Center (QBiC) Tübingen, University of Tübingen, Germany',
+            name: 'Susanne Jodoin',
+            affiliation: 'Quantitative Biology Center (QBiC) Tübingen, University of Tübingen, Germany',
             email: '',
             github: '@SusiJo',
             contribution: ['author'],
             orcid: '0009-0001-9152-7291'
         ],
         [
-            name: ' Maxime U Garcia',
+            name: 'Maxime U Garcia',
             affiliation: 'Seqera',
             email: 'maxime.garcia@seqera.io',
             github: '@maxulysse',
             contribution: ['maintainer'],
             orcid: '0000-0003-2827-9261'
         ],
+        [
+            name: 'Famke Bäuerle',
+            affiliation: 'Quantitative Biology Center (QBiC) Tübingen, University of Tübingen, Germany',
+            email: 'famke.baeuerle@uni-tuebingen.de',
+            github: '@famosab',
+            contribution: ['maintainer'],
+            orcid: '0000-0003-1387-0251'
+        ],
+        [
+            name: 'Gisela Gabernet',
+            affiliation: '',
+            email: '',
+            github: '@ggabernet',
+            contribution: ['contributor'],
+            orcid: ''
+        ],
+        [
+            name: 'Anders Sune Pedersen',
+            affiliation: '',
+            email: '',
+            github: '@asp8200',
+            contribution: ['contributor'],
+            orcid: ''
+        ],
+        [
+            name: 'Matilda Åslin',
+            affiliation: '',
+            email: '',
+            github: '@matrulda',
+            contribution: ['contributor'],
+            orcid: ''
```

```suggestion
            name: 'Matilda Åslin',
            affiliation: 'SNP&SEQ Technology Platform, Department of Medical Sciences, Uppsala University',
            email: 'matilda.aslin@medsci.uu.se',
            github: '@matrulda',
            contribution: ['contributor'],
            orcid: '0000-0002-2450-6415'
```
Some more information for my entry. Thanks for this!